label_studio/core/settings/base.py

```diff
@@ -468,10 +468,13 @@
 ALLOW_IMPORT_TASKS_WITH_UNKNOWN_EMAILS = get_bool_env('ALLOW_IMPORT_TASKS_WITH_UNKNOWN_EMAILS', default=False)
 
 """ React Libraries: do not forget to change this dir in /etc/nginx/nginx.conf """
+
+ENABLE_MONOREPO_ENV = get_bool_env('ENABLE_MONOREPO_ENV', default=False)
+
 # EDITOR = label-studio-frontend repository
-EDITOR_ROOT = os.path.join(BASE_DIR, '../../web/dist/libs/editor')
+EDITOR_ROOT = os.path.join(BASE_DIR, '../../web/dist/libs/editor') if ENABLE_MONOREPO_ENV else os.path.join(BASE_DIR, '../../label_studio/frontend/dist/lsf')
 # DM = data manager (included into FRONTEND due npm building, we need only version.json file from there)
-DM_ROOT = os.path.join(BASE_DIR, '../../web/dist/libs/datamanager')
+DM_ROOT = os.path.join(BASE_DIR, '../../web/dist/libs/datamanager') if ENABLE_MONOREPO_ENV else os.path.join(BASE_DIR, '../../label_studio/frontend/dist/dm')
 # FRONTEND = GUI for django backend
 REACT_APP_ROOT = os.path.join(BASE_DIR, '../../web/dist/apps/labelstudio')
```

Comment:
I assume in both cases we are just using the same App Shell location regardless of whether this is served as a monorepo.